manager/manager.go

```diff
@@ -954,13 +954,18 @@ func (m *Manager) becomeLeader(ctx context.Context) {
 		// store. Don't check the error because
 		// we expect this to fail unless this
 		// is a brand new cluster.
-		store.CreateCluster(tx, defaultClusterObject(
+		err := store.CreateCluster(tx, defaultClusterObject(
 			clusterID,
 			initialCAConfig,
 			raftCfg,
 			api.EncryptionConfig{AutoLockManagers: m.config.AutoLockManagers},
 			unlockKeys,
 			rootCA))
+
+		if err != nil {
+			log.G(ctx).WithError(err).Errorf("Error creating cluster object")
```

Comment:
let's make sure that this error contains the cluster id. Is there any other info we can log here?

Reply:
Which cluster id do you want to log ? The one that already exists ? A cluster object may not always exist.

Reply:
Which cluster id do you want to log ? The one that already exists ? A cluster object may not always exist.